profile.d/security

```diff
@@ -26,3 +26,8 @@ RELEASE_GIT_PRODUCTION_BRANCH=master
 #         as retrieved from ./utils/getJenkinsVersion.py
 
 MAVEN_REPOSITORY_PRODUCTION_NAME=releases
+
+RELEASE_GIT_BRANCH=security-stable-2.479
+MAVEN_REPOSITORY_NAME=canibus
+JENKINS_VERSION=2.479.2
+RELEASELINE=-stable
```

Comment:
```suggestion
RELEASELINE=-stable

```

nitpicking: https://thoughtbot.com/blog/no-newline-at-end-of-file ;)
POSIX is the law, this is the way